tutorials/intro-to-geospatial-analysis/index.md

```diff
@@ -0,0 +1,449 @@
+---
+title: An Introduction to Geospatial Analysis
+description: This workshop provides a fundamental overview of remote sensing and geospatial analysis using the JavaScript API of Earth Engine.
+author: TC25
+tags: introductory, GIS, remote sensing, Google Earth Engine
+date_published: 2019-10-09
+---
+
+[Geometries - Open In Code Editor](https://code.earthengine.google.com/9d0bd55b88c3588dd189c3b512378a0b)
+
+[Features - Open In Code Editor](https://code.earthengine.google.com/bb1980ff4b6ddab0262de00956f89dcb)
+
+[Images - Open In Code Editor - Geometries](https://code.earthengine.google.com/234ba0ab11dd82112602de05c0bc3bd8)
+
+We will introduce types of geospatial data, using these data on the Earth Engine platforms, and discuss a host of functionality to visualize and analyze them. This workshop was originally created for a workshop during Yale-NUS Data 2.0 hackathon and then updated for Yale GIS Day 2018. 
+
+## Introduction
+Collection, visualization, and analysis of geographical or spatial data.
+
+## Data types
+
+- Vector data represent lat-long coordinates
+
+- Raster data comprises of pixels with associated values
+
+---
+
+
+- Points
+
+![Points](points.png)
+
+- Lines
+
+![Lines](line-vector.png)
+
+- Polygons
+
+![Polygons](polygon-vector.png)
+
+- Raster layers/bands
+
+![Raster](map-algebra.png)
+
+Image sources: https://gisgeography.com/spatial-data-types-vector-raster/
+
+---
+
+## Google Earth Engine platform
+
+[Introductory video](https://www.youtube.com/watch?v=gKGOeTFHnKY&feature=youtu.be, "Video")
+
+[Code Editor](https://code.earthengine.google.com/ "Earth Engine Code Editor")
```

Comment:
Broken link?

Reply:
Seems to be working for me; you may need to already have access to GEE to reach the code editor through that link.

Reply:
I see:
```
[Code Editor](https://code.earthengine.google.com/ "Earth Engine Code Editor")
```
I think you meant:
```
[Code Editor](https://code.earthengine.google.com/)
```